gui/default/syncthing/core/notifications.html

```diff
@@ -116,3 +116,25 @@ <h3 class="panel-title"><span class="fas fa-bolt"></span>&ensp;<span translate>A
     </div>
   </div>
 </notification>
+
+<notification id="authenticationUserAndPassword">
+  <div class="panel panel-success">
+    <div class="panel-heading">
+      <h3 class="panel-title"><span class="fas fa-bolt"></span>&nbsp;<span translate>GUI Authentication: Set User and Password</span></h3>
+    </div>
+    <div class="panel-body">
+      <p>
+      <span translate>Username/Password has not been set for the GUI authentication. Please consider setting it up.</span>
```

Comment:
```suggestion
      <span translate>Username/Password has not been set for GUI authentication. Please consider setting it up.</span>
```
Feels more natural to me. And generally I'd add a short explanation of why you want to set up authentication - there's good reasons for both choices. Something like e.g. "If you want to prevent other users on this computer to access Syncthing and through it your files, consider setting up authentication." (which is maybe not a good sentence, but hopefully illustrates what I mean).

Reply:
Done. I used your sentence, since I don't have any better idea how to illustrate this